### PR TITLE
overlay.d/15fcos: update link to FAQ

### PR DESCRIPTION
**Pull request by dustymabe**
Also one wording change.

### CODE REVIEW

File: overlay.d/15fcos/usr/libexec/coreos-check-wireless-firmwares [hunk 1]
```diff
@@ -49,12 +49,12 @@ motd_path=/run/motd.d/30_wireless_firmwares_warning.motd
...
 ${warn}
 ##########################################################################
-WARNING: The NetworkManager-wifi is a requested layered package on this
+WARNING: NetworkManager-wifi is a requested layered package on this
 system, but no Wi-Fi drivers are requested. The Wi-Fi drivers will no
 longer be included by default in the future.
 
 More context and remediation steps are available in the following FAQ entry:
-https://docs.fedoraproject.org/en-US/fedora-coreos/sysconfig-enabling-wifi/
+https://docs.fedoraproject.org/en-US/fedora-coreos/faq/#_how_do_i_keep_dropped_wireless_firmware
```
Comment (dustymabe) [inline overlay.d/15fcos/usr/libexec/coreos-check-wireless-firmwares]: do we think the link here is too long?

Comment (jlebon) [reply overlay.d/15fcos/usr/libexec/coreos-check-wireless-firmwares]: I guess if you consider that people seeing this on the VGA console and not serial console will have to retype it, yeah a bit. Looks like Asciidoctor supports customizing the anchor name pretty easily: https://github.com/coreos/fedora-coreos-docs/pull/635

Comment (dustymabe) [reply overlay.d/15fcos/usr/libexec/coreos-check-wireless-firmwares]: pushed an update!

**Review by jbtrystram — APPROVED** [review]
Lgtm

**Pull request merged**